.github/workflows/ci.yaml

```diff
@@ -39,29 +39,22 @@ jobs:
 
       - uses: actions/setup-node@v1
         with:
-          node-version: 16.13
+          node-version: 16.15
 
       - uses: actions/cache@v2
-        id: deps-cache
+        id: mix-cache
         with:
-          path: deps
-          key: ${{ runner.os }}-deps-${{ hashFiles(format('{0}/mix.lock', github.workspace)) }}
-          restore-keys: |
-            ${{ runner.os }}-deps-
-
-      - uses: actions/cache@v2
-        id: build-cache
-        with:
-          path: _build
-          key: ${{ runner.os }}-build-${{ matrix.otp-version }}-${{ matrix.elixir-version }}-${{ hashFiles(format('{0}/mix.lock', github.workspace)) }}
+          path: |
+            deps
+            _build
+            priv/plts
+          key: ${{ runner.os }}-mix-${{ matrix.otp-version }}-${{ matrix.elixir-version }}-${{ hashFiles(format('{0}/mix.lock', github.workspace)) }}
 
       - uses: actions/cache@v2
         id: npm-cache
         with:
           path: ~/.npm
           key: ${{ runner.os }}-npm-${{ hashFiles(format('{0}/assets/package-lock.json', github.workspace)) }}
-          restore-keys: |
```

Comment:
This is not valid?

Reply:
If I remember right, this was not useful… I'll have to go back to the documentation!

Reply:
> **Optional** A string containing alternative restore keys, with each restore key placed on a new line. If no cache hit occurs for key, these restore `keys` are used sequentially in the order provided to find and restore a cache. 

https://docs.github.com/en/actions/using-workflows/caching-dependencies-to-speed-up-workflows#input-parameters-for-the-cache-action